src/common/stream_status.rs

```diff
@@ -7,7 +7,7 @@ pub enum StreamStatus {
     /// StreamStatusNoData indicates that the audio data stream contains no data.
     StreamStatusNoData = 1,
 
-    /// StreamStatusNoData indicates that the audio data stream contains no data.
+    /// StreamStatusNoData indicates that the audio data stream contains partial data of a speak request.
```

Comment:
typo here, should start with  **StreamStatusPartialData**

Reply:
Done.